src/amo/containers/App.js

```diff
@@ -17,30 +19,32 @@ export class AppBase extends React.Component {
     i18n: PropTypes.object.isRequired,
     isAuthenticated: PropTypes.bool,
     location: PropTypes.object.isRequired,
+    MastHeadComponent: PropTypes.node,
+  }
+
+  static defaultProps = {
+    MastHeadComponent: MastHead,
   }
 
   accountButton() {
     const { handleLogIn, i18n, isAuthenticated, location } = this.props;
-    if (isAuthenticated) {
-      return <p>You are logged in</p>;
-    }
     return (
-      <button className="button" onClick={() => handleLogIn(location)}
+      <button className="button AccountButton" onClick={() => handleLogIn(location)}
               ref={(ref) => { this.logInButton = ref; }}>
-        {i18n.gettext('Log in')}
+        <span>{ isAuthenticated ? i18n.gettext('Logout') : i18n.gettext('Login/Sign up') }</span>
```

Comment:
Should be "Log out" and "Log in".

I'll just leave this here... http://loginisnotaverb.com/ 😉